README.md

```diff
@@ -271,6 +311,48 @@ To clean up the stopped containers, run:
 docker compose down --volumes
 ```
 
+#### Using Podman
+
+If you're using Podman, you can use either `podman-compose` or `docker compose` (thanks to the compatibility layer). Here are both options:
+
+Using podman-compose directly:
+```bash
+# Start the application
+podman-compose up --build
```

Comment:
hm is the dash always necessary or impact things compared to "podman compose up --build"? I usually do podman compose up --build